### PR TITLE
Change RxJs version. Update test.

### PR DESCRIPTION
**Pull request by Sciumo**
Very minor change to version and update to test. Some Visual Code cruft.

### CODE REVIEW

**Comment by hellosmithy** [comment]
Looks fine to me - kind of weird having comments in the json file though. I don't use vscode but does it parse that ok?

File: test/chai-rx.spec.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
+import chai from 'chai';
 import Rx from 'rx';
 const { onNext, onError, onCompleted } = Rx.ReactiveTest;
 
 describe('chai-rx', () => {
   describe('emit', () => {
 
     describe('`expect` syntax', () => {
+      console.log('expect');
```
Comment (hellosmithy) [inline test/chai-rx.spec.js]: Can you remove this `console.log`?

**Review by hellosmithy — CHANGES_REQUESTED** [review]

**Comment by hellosmithy** [comment]
Thinking about it, I'm not sure about adding in editor specific launch.json file here. It makes sense on a big team project where lots of people need to share their config. But this is a very simple package that just has a couple of scripts. You can always maintain the launch.json file out of version control if you really want it

File: test/chai-rx.spec.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
       it('should decorate `expect().to`', () => {
         const emit = expect().to.emit;
-        expect(emit).to.be.a.function;
+        expect(emit).to.be.a('function');
```
Comment (hellosmithy) [inline test/chai-rx.spec.js]: Good catch 👍

**Comment by Sciumo** [comment]
Ok.

**Pull request merged**